packages/dashboard/src/components/Card/Chart/Chart.tsx

```diff
@@ -5,20 +5,56 @@ import { LineChart } from '@uifabric/charting/lib/LineChart';
 import { VerticalBarChart } from '@uifabric/charting/lib/VerticalBarChart';
 import { DonutChart } from '@uifabric/charting/lib/DonutChart';
 import { PieChart } from '@uifabric/charting/lib/PieChart';
-import { StackedBarChart } from '@uifabric/charting/lib/StackedBarChart';
+import {
+  StackedBarChart,
+  IDataPoint,
+  ILegendDataItem,
+  MultiStackedBarChart
+} from '@uifabric/charting/lib/StackedBarChart';
 
 export class Chart extends React.Component<IChartInternalProps, {}> {
+  public static defaultProps = {
+    compactChartWidth: 250
+  };
+
+  private _chartLabel: string | undefined;
+  private _isMultiBarChart = false;
+  private _colors: string[] | undefined;
+  private _singleChartDataPoints: IDataPoint[] | undefined;
+
+  public constructor(props: IChartInternalProps) {
+    super(props);
+
+    if (props.chartLabels && props.chartLabels.length === 1) {
+      this._chartLabel = props.chartLabels[0];
+    }
+
+    if (props.data) {
+      this._isMultiBarChart = props.data.length > 1;
+    }
+
+    this._singleChartDataPoints = props.dataPoints;
+
+    if (this._isMultiBarChart === false && props.data && props.data.length > 0) {
+      this._singleChartDataPoints = props.data[0];
+    }
+
+    if (props.legendColors && props.legendColors.length > 0) {
+      this._colors = props.legendColors.map((item: ILegendDataItem) => item.legendColor);
+    }
+  }
+
   public render(): JSX.Element {
     switch (this.props.chartType) {
       case ChartType.VerticalBarChart: {
         return (
           <VerticalBarChart
-            data={this.props.dataPoints}
+            data={this._singleChartDataPoints}
```

Comment:
>_singleChartDataPoints [](start = 23, length = 22)

why are we changing this? I am not sure I understand this implementation. @Karthik, can you please take a look